front/src/modules/timesStops/utils.ts

```diff
@@ -43,3 +47,36 @@ export const formatSuggestedViasToRowVias = (
     };
   });
 };
+
+export function findNextScheduledPoint(
+  operationalPoints: OperationalPointWithTimeAndSpeed[],
+  tsPathPointsWithOpPointIndices: PathPointOpPointCorrespondance[],
+  sugOpIndex: number
+) {
+  const pathIndex = tsPathPointsWithOpPointIndices.findIndex(
+    (pathPoint) => pathPoint.correspondingOpPointIndex === sugOpIndex
+  );
+  const nextPathPoint = tsPathPointsWithOpPointIndices[pathIndex + 1];
+  return operationalPoints[nextPathPoint?.correspondingOpPointIndex];
+}
+
+const getDigits = (u: string | undefined) => (u === 's' || u === '%' ? 0 : 1);
+
+export function formatDigitsAndUnit(value: string | undefined, unit?: string) {
+  if (value === undefined || value === 'none') {
+    return '';
+  }
+  if (marginRegExValidation.test(value)) {
+    const [, v, , u] = value.match(marginRegExValidation) || [];
```

Comment:
Why not directly match like in useComputedMargins ?
```suggestion
  const valueSplit = value.match(marginRegExValidation);
  if (valueSplit) {
    const [, v, , u] = valueSplit;
```

Reply:
ah yes, done